proto/cmp/types/v1alpha/seat_map.proto

```diff
@@ -0,0 +1,154 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+
+// Basic representation of a seat with optional features.
+message Seat {
+  // Unique identifier, e.g., "12B", "A26"
+  string seat_id = 1;
+
+  // Seat location
+  SeatMapLocation location = 2;
+
+  // Static seat features
+  repeated SeatAttribute features = 3;
+
+  // Restrictions
+  repeated SeatAttribute restrictions = 4;
+}
+
+// FIXME: Can we make this a big enum?
+message SeatAttribute {
+  // Static attributes to be used for features and restrictions
+  string name = 1;
+
+  // Human readable description
+  string description = 2;
+
+  // Value to be used for restrictions or conditions like "min_age"
+  int32 value = 3;
+}
+
+enum AreaType {
+  AREA_TYPE_UNSPECIFIED = 0;
+  AREA_TYPE_RECTANGLE = 1;
+  AREA_TYPE_CIRCLE = 2;
+  AREA_TYPE_POLYGON = 3;
+}
+
+// Image location for seat maps with bitmap images. This info is generallt used with
+// HTML `area` tag.
+message BitmapSeatLocation {
+  AreaType type = 1;
+  repeated int32 coordinates = 2;
+}
+
+// Vector Seat Location for SVG
+message VectorSeatLocation {
+  // Label, e.g. "section-TERRACE-26-34-2-label" for SVG
+  string svg_label = 1;
+}
+
+// Representation of a seat location in SVG or bitmap.
+message SeatMapLocation {
+  // Seat location it should be one of vector (SVG) or bitmap location message
+  // types.
+  oneof location {
+    VectorSeatLocation vector = 1;
+    BitmapSeatLocation bitmap = 2;
+  }
+}
+
+// Section could be a block of rows or a specific area in a venue like a stage, or a
+// section in a plane.
+message Section {
+  // Level, section or row identifier, e.g., "Upper", "Balcony" or "Section 101",
+  // "Orchestra" or "A", "12".
+  string id = 1;
```

Comment:
For maps it's `map_id`, for seat it's `seat_id`, for section it's `id`?
We should have it consistent. I'm in favor of getting rid of the prefix in case it's inside of the message the id is for (which is the same for other fields like name).

Reply:
I don't know how that one slipped, I was using `<name>_id` for all. But yes, I'm also in favor of loosing the prefix.